### PR TITLE
Issue 4 delete an entry

### PR DESCRIPTION
**Pull request by katiefugate**
Screen Shot 2021-04-01 at 3 29 27 PM
Screen Shot 2021-04-01 at 3 29 45 PM

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -107,3 +108,77 @@ ul {
...
+  font-size: 1rem;
+}
+
+@media screen and (min-width: 768px) {
+  .modalContainer {
+    width: 25rem;
+    top: 50%;
+    left: 50%;
+    margin-top: -3.5rem;
+    margin-left: -12.5rem;
```
Comment (codingcodymiller) [inline css/styles.css]: WOAH that is a lot of margin adjustment, and especially bad, since it is a negative value. Why is this necessary? If you want to use negative amrgin, that probably means that another element just needs a smaller margin. Is this to center the element? If so, you can use `transform: translate(-50%, -50%)`. The problem here is this:
```
    top: 50%;
    left: 50%;
```
This code will center the _top left corner_ of the element, not the center, so it ends up being off-center. The line I gave you shifts the point that is being centered to the center of the element, by moving the origin point backwards by 50% of the height and width of the element. Now the top left corner won't be centered, it will be the center point of the element.

File: js/main.js [hunk 2]
```diff
@@ -140,3 +154,51 @@ function editHandler(event) {
...
+  data.view = 'entries';
+  overlay.className = 'overlay hidden';
+  modal.className = 'modalContainer';
+  entriesContainer.className = 'container current';
+  formContainer.className = 'container hidden';
+  var liList = document.querySelectorAll('.entry');
+  for (var i = 0; i < data.entries.length; i++) {
+    if (currentEditId === data.entries[i].entryId) {
+      data.entries.splice(i, 1);
+      liList[i].replaceWith('');
```
Comment (codingcodymiller) [inline js/main.js]: You can just use `remove()` here instead of `replaceWith`

File: js/main.js [hunk 3]
```diff
@@ -82,7 +82,9 @@ function addEntry(entry) {
 
 function contentLoadHandler(event) {
   for (var i = 0; i < data.entries.length; i++) {
-    ul.appendChild(addEntry(data.entries[i]));
+    if (data.entries[i] !== null) {
```
Comment (codingcodymiller) [inline js/main.js]: Why is this necessary? Do you have `null` in your array of entries?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 3 above)
Comment (katiefugate) [reply js/main.js]: I did somehow but I removed it, I forgot to take this part out after.

**Review by codingcodymiller — APPROVED** [review]
